cpp/tests/io/parquet_experimental_reader_test.cpp

```diff
@@ -134,7 +134,7 @@ auto create_parquet_with_stats()
 
   cudf::io::write_parquet(out_opts);
 
-  return std::pair{std::move(table), buffer};
+  return std::pair{std::move(table), std::vector<uint8_t>(buffer.begin(), buffer.end())};
```

Comment:
Ah, host buffer sink can only be vector<char>. Hopefully this isn't expensive.

Reply:
Yeah, hoping the same as well. Or we can keep the cast to `uint8_t`

Reply:
Reverting the cast